recyclerviewfastscroller/src/main/java/xyz/danoz/recyclerviewfastscroller/calculation/progress/VerticalLinearLayoutManagerScrollProgressCalculator.java

```diff
@@ -26,6 +26,8 @@ public float calculateScrollProgress(RecyclerView recyclerView) {
         int lastFullyVisiblePosition = layoutManager.findLastCompletelyVisibleItemPosition();
 
         View visibleChild = recyclerView.getChildAt(0);
+        if (visibleChild == null)
+            return 0;
```

Comment:
Ahh, interesting. But it's only recommended/forced for android source. I will still change it, wait a moment..